app/controller.py

```diff
@@ -188,3 +195,51 @@ def troll_katie():
         turn_light_off()
         sleep(0.13)
     return redirect(('/' + get_current_day_name()).lower())
+
+"""
+Made initially for testing sound.
+Use it now and then to check if everything working properly.
+"""
+@app.route('/sound')
+def yolo_def():
+    from time import sleep
+    print os.getcwd()
+    cmd = ['mpc', 'update']
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE)
+    cmd = ['mpc', 'clear']
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE)
```

Comment:
I see a lot of these lines copy and pasted; we should consider making a helper function instead:

```python

def run_command(args):
  subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE)

# Then the rest of the code becomes a little more DRY:

@app.route('/sound')
def play_sound():
    run_command(['mpc', 'update'])
    run_command(['mpc', 'clear'])
    run_command(['mpc', 'add', alarm_file])
    run_command(['mpc', 'play'])
    sleep(5)
    run_command(['mpc', 'stop'])
    return redirect(('/' + get_current_day_name()).lower())
"""